lib/components/molecules/Form/DDFileInput/DDFileInput.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+import React, { useState } from 'react';
+import type { Node } from 'react';
+import { Field } from 'formik';
+import styled from 'styled-components';
+
+import Label from '../../../atoms/Label';
+import Input from '../../../atoms/FileInput';
+import styles from './DDFileInput.style';
+import useOpenFileDialog from '../../../../core/utils/useOpenFileDialog';
+
+type Props = {
+  id: string,
+  className?: string,
+  type?: string,
+  name: string,
+  label: Node,
+  labelProps?: Object,
```

Comment:
Assigning a object or Array data type is very open ended. Here you should import the types from Label component.